lib/trailblazer/operation/policy.rb

```diff
@@ -2,6 +2,21 @@
 
 module Trailblazer
   class NotAuthorizedError < RuntimeError
+    attr_reader :query, :record, :policy
+
+    def initialize(options = {})
+      if options.is_a? String
```

Comment:
Also this options.fetch thing is from Ruby 1.9 right? I should refactor this into keyword args. Lol